docs/pages/es/architecture.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: Arquitectura de Meshery 
+permalink: es/architecture
+---
+
+## Arquitectura
+
+#### La arquitectura de Meshery puede ser vista desde dos perspectivas:
+ 
```

Comment:
Will you incorporate the [Deployments section](https://meshery.layer5.io/docs/architecture#deployments), too? 

Meshery se implementa como un conjunto de contenedores. Los contenedores de Meshery se pueden implementar en Docker o Kubernetes.